networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -36,6 +36,8 @@
 SWITCHGROUP_ROLE_APOD = 'ap'
 SWITCHGROUP_ROLE_NETPOD = 'np'
 
+DEFAULT_VLAN_RANGES = ["2000:3750"]
```

Comment:
Sounds a bit brutal to completely hardcode this. Should we maybe make this configurable and set this as default?
I acknowledge that all of this is pretty much tailored to our needs, but until now we tried to keep things in a way that somebody else could take this as a base, throw in their own values and run it. Maybe it's time for me to part ways with this idea - at least with the config generator.

Reply:
Are we yet at the point where we need a config for the config generator? ;) Maybe we are. Initially I thought about having this as a argument in `argparse`. But you might be right with that if somebody else would want to use it they'd have to write their own config generator (good thing we have most of the sanity checks inside the DriverConfig as validators). Let's keep it this way then.

Reply:
On that note, the netbox url is also hardcoded already :|

Reply:
That settles that :D